src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.spec.ts

```diff
@@ -214,19 +215,19 @@ describe('InstructorSessionEditPageComponent', () => {
     feedbackSessionName: 'test session',
     instructions: 'Instructions',
 
-    submissionStartTime: { hour: 23, minute: 59 },
+    submissionStartTime: getLatestTimeFormat(),
     submissionStartDate: { year: 0, month: 0, day: 0 },
```

Comment:
This line seems to be left out of the refactoring, is it intentional?

Reply:
```suggestion
    submissionStartDate: getDefaultDateFormat(),
```
@tau-bar Do make the above suggested change!